data/human/boarding missions.txt

```diff
@@ -150,9 +150,9 @@ phrase "assist merchant 3"
 	word
 		`captain `
 	word
-		`gives you`
-		`offers you`
-		`rewards you with`
+		`gives you `
+		`offers you `
+		`rewards you with `
```

Comment:
I don't believe these are necessary. At least, I didn't notice any issues with the spacing for this portion of the sentence.

Reply:
Looks fine without these changes:
<img width="558" alt="image" src="https://github.com/endless-sky/endless-sky/assets/98854112/24d73d5d-d8db-4bd7-86ac-629adde1da3c">

Reply:
after revisiting this section, I agree, I'll remove them from this push at the moment, and do some testing with the altered file.